### ISSUE TITLE
Update documentation for `TX_FIFO` variables

### ISSUE THREAD

**Issue by Mellich** [pc2/Aurora-HLS#32]
This issue is a follow up of #31. The `TX_FIFO_*` variables can be configured from the Makefile, but the effect of these variables is not fully clear. The documentation should be extended to better explain the effect of these variables on the functionality and resource utilization of the Aurora kernel.

**Comment by papeg** [comment]
I first thought about completely removing the threshold signals for TX, because they don't really serve a functional purpose. But they are part of the fifo status signals which can be read from the host code: https://github.com/pc2/Aurora-HLS/blob/2e4a05e22e9d5167c9f175b91dfc65a8e2f621e7/rtl/aurora_hls.v.template.v#L270

So they can be useful for checking the status of the internal workings. For the most cases only full and empty signals can be enough, but they give a little more detailed view, when configured correctly. It is still an option to remove them, to save some resources, but i think this effect would be very minimal. 

So in conclusion, i would not change the design and just update the docs as proposed.
